### PR TITLE
@alloy => Fix: hide rails when empty.

### PR DESCRIPTION
**Pull request by dblock**
I don't know what I am doing so please tell me if this is a bad idea.

This fixes a cosmetic problem related to https://github.com/artsy/force/issues/1060. If you go to an empty page like https://www.artsy.net/artist/munan-ovrelid/related-artists we will no longer show an empty title.

### Before

screen shot 2017-04-05 at 1 51 31 pm

### After

screen shot 2017-04-05 at 1 51 51 pm

### CODE REVIEW

**Comment by alloy** [comment]
Code-wise this seems good to me, I'm only wondering if the content below shouldn't move up in the case the rail is not shown? /cc @briansw

**Comment by dblock** [comment]
I don't think so cause that's a popup for my stuff.

**Comment by alloy** [comment]
What do you mean by “my stuff”?

I’d expect this over a large blank space:

screen shot 2017-04-05 at 20 30 39

**Comment by dblock** [comment]
This turned out to be more involved. There're two parts to every subview here, a specific part (eg. cv or articles) and artist's works. I refactored the shared piece into `ArtistArtworksView`. I am not sure it's right way to do that. Thoughts?

**Comment by alloy** [comment]
That looks right to me, but I can’t say if that is the idiomatic way it should be done in classic Force code. @craigspaeth do you have any objections?

**Comment by craigspaeth** [comment]
This is a solid way to accomplish this—no objections from me.

FWIW I can mention that personally I try avoid class inheritance whenever possible ([Gorilla & Banana problem and all that](https://medium.com/@cscalfani/goodbye-object-oriented-programming-a59cda4c0e53)) and might choose to instead compose the views "up a level" so they don't have to be repeated. e.g. the `ArtistArtworksView` could be rendered alongside the current view [in the router](https://github.com/artsy/force/blob/master/desktop/apps/artist/client/router.coffee#L60). That said, this is a pretty shallow + innocent class inheritance, so I'm not at all opposed to having it 👍

**Comment by alloy** [comment]
Alright, let's go for it then 👍

File: desktop/apps/artist/client/views/cv.coffee [hunk 1]
```diff
@@ -23,24 +21,10 @@ module.exports = class CVView extends Backbone.View
...
-    rail.collection.trigger 'sync'
-
   render: (artist = {}) ->
     artistMetadata = artistHelpers.artistMeta @model.toJSON()
     @$el.html template _.extend { showHelpers, artistMetadata }, artist
-    _.defer => @postRender()
-    this
-
-  remove: ->
+    if artistMetadata.length == 0
```
Comment (mzikherman) [inline desktop/apps/artist/client/views/cv.coffee]: This is causing a bug w/ missing CV's in production. `artistMetadata` can be blank (missing hometown/location), but they can still have a CV.

**Pull request merged**